inprocess/build.gradle

```diff
@@ -0,0 +1,65 @@
+plugins {
+    id "java-library"
+    id "java-test-fixtures"
+    id "maven-publish"
+
+    id "me.champeau.jmh"
```

Comment:
Delete.

Reply:
deleted but not deleting for util (since util has jmh)